Extension/tscCompileList.txt

```diff
@@ -0,0 +1,13 @@
+# This file is consumed in tools\prepublish.js
+# '#' at the start of the line is treated as a comment and is ignored.
+# whitespace empty lines are also ignores.
+#
+# This file is to be used if you do not wish to webpack a .ts file
+# If you do not want it to be webpacked:
+# 1. Please make sure it is not in the src directory
```

Comment:
oh... I was thinking we would want to move the settings UI into the src folder at some point.  Not a big deal, I guess, but we can revisit this when we make that call.

Reply:
IMO and with the addition of UI, we should move towards more of a [webpage](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files) styled folder structure. 

E.g.

dist/
   main.js
   *.html
   images/
   styles/
   scripts/